### DISCUSSION TITLE
A small gift for all contributors

### DISCUSSION THREAD

**Discussion by HFO4** [cloudreve/cloudreve#1344] [Show and tell]
Hi all,

![image](https://user-images.githubusercontent.com/16058869/172848021-cde5bc05-7ba5-4e29-8724-1f0426cecb18.png)

We have prepared a batch of acrylic Cloudreve keychains, which will be given away for free by mail to you (China mainland only).
If you have contributed to Cloudreve related repos (backend, frontend, docs, ...), feel free to fill the form: https://forms.office.com/r/JdeLz5p53y

---
Chinese version: https://forum.cloudreve.org/d/2382

---

Below we well mention contributors, sorry about the disturb😁

@WeidiDeng 
@vvisionnn 
@hiCasper 
@mritd 
@topjohncian 
@ihipop 
@milkice233 
@XYenon 
@kaaass 
@Cyanoxygen 
@kleinsea 
@xuchaoa 
@sb-child 
@cha0sCat 
@yeungc 
@kikoqiu 
@GuerraMorgan 
@Ink-33 
@breezechen 
@KusakabeShi 
@zjsth92 
@byxiaozhi 
@xb2016

**Comment by XYenon** [comment]
Get! Thanks!
![DSC_0115](https://user-images.githubusercontent.com/20698483/173298173-81ef6b4c-1145-48ae-a789-d44ed1e80bf3.JPG)

**Comment by sb-child** [comment]
![IMG_20220616_065335](https://user-images.githubusercontent.com/55868015/173956335-40095732-62a9-4c3e-b952-86c23a330af2.jpg)

received, thank you!

**Comment by caspershw** [comment]
Thanks!
![](https://s3.bmp.ovh/imgs/2022/07/09/795880b04abe852b.png)
